planning/behavior_path_planner/src/scene_module/pull_over/pull_over_module.cpp

```diff
@@ -229,7 +229,8 @@ void PullOverModule::onFreespaceParkingTimer()
 
   const bool is_new_costmap =
     (clock_->now() - planner_data_->costmap->header.stamp).seconds() < 1.0;
-  if (isStuck() && is_new_costmap) {
+  constexpr double path_update_duration = 1.0;
```

Comment:
If it takes longer to engage and the path is changed during the driving, this value should be increased.